packages/social-previews/src/facebook-preview/post-preview.tsx

```diff
@@ -11,22 +11,37 @@ export const FacebookPostPreview: React.FC< FacebookPreviewProps > = ( {
 	url,
 	user,
 	customText,
-	customImage,
+	media,
 	imageMode,
 } ) => {
-	const [ mode, isLoadingImage, imgProps ] = useImage( { mode: imageMode } );
+	const [ mode ] = useImage( { mode: imageMode } );
 	const modeClass = `is-${ mode === PORTRAIT_MODE ? 'portrait' : 'landscape' }`;
 
 	return (
 		<div className="facebook-preview__post">
 			<FacebookPostHeader user={ user } />
 			<div className="facebook-preview__content">
 				{ customText && <CustomText text={ customText } url={ url } forceUrlDisplay /> }
-				<div className={ `facebook-preview__body ${ isLoadingImage ? 'is-loading' : '' }` }>
-					<div className={ `facebook-preview__custom-image ${ modeClass }` }>
-						{ /* eslint-disable jsx-a11y/alt-text */ }
-						<img src={ customImage } { ...imgProps } />
-					</div>
+				<div className="facebook-preview__body">
+					{ media ? (
+						<div className={ `facebook-preview__media ${ modeClass }` }>
+							{ media.map( ( mediaItem, index ) => (
+								<div
+									key={ `facebook-preview__media-item-${ index }` }
+									className={ `facebook-preview__media-item ${ modeClass }` }
+								>
+									{ mediaItem.type.startsWith( 'video/' ) ? (
+										// eslint-disable-next-line jsx-a11y/media-has-caption
+										<video controls>
+											<source src={ mediaItem.url } type={ mediaItem.type } />
+										</video>
+									) : (
+										<img alt={ mediaItem.alt || '' } src={ mediaItem.url } />
+									) }
+								</div>
+							) ) }
+						</div>
+					) : null }
```

Comment:
What do you think of using `&&` for all these conditional JSX elements across the files? That way we do not need the `null`.

```
{ media && ( <div>....</div> ) }
```

Reply:
I usually prefer to avoid `&&` if I am not sure the about the value.

For example if `media` is `0`, then `media && ...` will render `0` instead of nothing.

Well, sometimes, I do use `&&` to make the code look better lol.